manager/manager.go

```diff
@@ -37,21 +37,24 @@ import (
 )
 
 type Manager struct {
-	nornsDispatch    map[string]nornDispatcher
+	nornFilter       map[string]filterDispatcher
```

Comment:
We should have two maps - one of norn ids to filterers, and one of endpoint urls to dispatchers.